app/src/main/res/layout-land/profile_chooser_profile_view.xml

```diff
@@ -37,13 +37,9 @@
 
       <com.google.android.material.imageview.ShapeableImageView
         android:id="@+id/profile_avatar_img"
-        android:layout_width="72dp"
-        android:layout_height="72dp"
-        android:layout_marginBottom="8dp"
-        app:shapeAppearanceOverlay="@style/ShapeAppearanceOverlay.RoundedShape"
-        app:strokeColor="@color/component_color_profile_chooser_activity_avatar_border_color"
-        app:strokeWidth="1dp"
-        profile:src="@{viewModel.profile.avatar}" />
+        style="@style/ShapeAbleImageView"
```

Comment:
Please format this whole file using shortcut as suggested in last review. It not format yet.

Reply:
Done @MohitGupta121